src/Visualization.jl

```diff
@@ -8,15 +8,14 @@ function Makie.plot(tn::TensorNetwork{A}; kwargs...) where {A}
     f = Makie.Figure()
 
     p, ax = Makie.plot!(f[1, 1], tn; kwargs...)
-    # display(f)
+    display(f)
 
-    # return f, ax, p
-    return f
+    return f, ax, p
```

Comment:
This code gave me problems like this. That's way I left it the other way.

Reply:
Are you sure of the call to `display`? Other `plot` methods do not return the `ax` object.

Reply:
I always have found that `plot` returns `f, ax, p` (for example [here](https://github.com/MakieOrg/Makie.jl/blob/2160439bae753e54d306b6e12d305cebe5763d4e/test/statistical_tests.jl#LL41) and [here](https://github.com/MakieOrg/GraphMakie.jl/blob/85de50648d15b376434a28c6d7471c9d92f14ba0/docs/examples/plots.jl#LL140)). The `ax` is useful because it enables the user to modify the axis of the plot.
The `display` is necessary since without it you would have to do first `f, ax, p = plot(...)` and then manually `display(f)` to see the results plotted.

Reply:
Okay, I see that `f, ax, p` on `plot` functions but on `plot!` functions, only `p` is returned (currently `p` and `ax` are returned).

I'm still not sure about the `display`. `GraphMakie` does not call `display` for example.

Reply:
Okay, we can remove it then.

Reply:
Do you know if other `plot` functions in `Makie` are automatically displayed without calling to `display`?

Reply:
I tested it now and only the exclamation mark functions (e.g., `plot!`) are displayed automatically, the others don't.

Reply:
Sorry, I deleted the last comment since I said it the other way around: Only the exclamation mark functions (e.g., `plot!`) are not displayed automatically, the others are. So we actually do need the `display` in `plot`, it is not necessary for `plot!`. I will change it as it was.

Reply:
And how does other `Makie` projects display? Do they call `display`? We have to inspect the source code.

Reply:
Okay, this makes sense: they return `FigureAxisPlot` on `plot` and `AxisPlot` on `plot!`.

> ... We could do the same calling by `graphplot` instead of `graphplot!`, but we would need to refactor the code so we could not use exactly the same code for our `plot` and `plot!` functions. ...

Great. In order to not duplicate the code, try to abstract common code.

Reply:
Now the `plot` functions return the desired structs. I have combined the code into some functions, so we don't duplicate the code.